code/modules/antagonists/ert/ert.dm

```diff
@@ -189,3 +194,99 @@
 
 	missiondesc += "<BR><B>Your Mission</B> : [ert_team.mission.explanation_text]"
 	to_chat(owner,missiondesc)
+
+
+/datum/antagonist/ert/families
+	name = "Space Police Responder"
+	antag_hud_type = ANTAG_HUD_SPACECOP
+	antag_hud_name = "hud_spacecop"
+
+/datum/antagonist/ert/families/apply_innate_effects(mob/living/mob_override)
+	..()
+	var/mob/living/M = mob_override || owner.current
+	add_antag_hud(antag_hud_type, antag_hud_name, M)
+
+
+/datum/antagonist/ert/families/remove_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
+	remove_antag_hud(antag_hud_type, M)
+	..()
+
+/datum/antagonist/ert/families/greet()
+	to_chat(owner, "<B><font size=3 color=red>You are the [name].</font></B>")
+
+	var/missiondesc = "After an uptick in gang violence on [station_name()], you are responding to emergency calls from the station for immediate SSC Police assistance!\n"
+	missiondesc += "<BR><B>Your Mission</B>:"
+	missiondesc += "<BR> <B>1.</B> Secure the situation and crack down on any gang activity. You can view gangsters with your sunglasses."
+	missiondesc += "<BR> <B>2.</B> There is an undercover police officer on station. Secure him, receive his intel, and extract him safely."
+	missiondesc += "<BR> <B>3.</B> Minimize civilian casualties, but defend yourself and civilians from hostile gangsters."
+	missiondesc += "<BR> <B>3.</B> If Security is found to be violating the rights of citizens, detain them as per your authority as Spinward Stellar Coalition officers."
```

Comment:
*non-gangster citizens

Reply:
*as a Spinward Stellar Coalition officer.

Reply:
intentional, gangsters are still citizens of the SSC